packages/phone-number-privacy/package.json

```diff
@@ -6,17 +6,23 @@
   "license": "Apache-2.0",
   "main": "dist/index.js",
   "scripts": {
-    "serve": "firebase serve --only functions",
-    "deploy": "firebase deploy --only functions",
+    "dev": "yarn build && firebase serve --only functions",
+    "deploy:staging": "firebase deploy --only functions --project celo-phone-number-privacy-stg",
+    "deploy:prod": "firebase deploy --only functions --project celo-phone-number-privacy",
+    "config:get:staging": "firebase functions:config:get --project celo-phone-number-privacy-stg",
+    "config:get:prod": "firebase functions:config:get --project celo-phone-number-privacy",
     "clean": "tsc -b . --clean",
     "build": "tsc -b .",
     "lint": "tslint --project .",
-    "test": "jest --runInBand"
+    "test": "jest --runInBand",
+    "db:migrate": "NODE_ENV=development ts-node scripts/runMigrations.ts"
```

Comment:
it looks like in dev mode, this migrates to `host: 'fakeHost'.  so does this even create anything?  if so which environment (locally vs stg vs prod)

Reply:
Yeah you would have to populate those values with the ones from our staging DB but didn't want those checked in.

Reply:
kk makes sense